bootstrap.php

```diff
@@ -19,6 +22,64 @@
 	}
 	return;
 }
+
+/*
+ * Initialize coming soon module via container
+ */
+$nfd_module_container = new Container();
+// Set plugin to container
+$nfd_module_container->set(
+	'plugin',
+	$nfd_module_container->service(
+		function() {
+			return new Plugin(
+				array(
+					'id'   => 'web',
+					'file' => WEB_PLUGIN_FILE,
+				)
+			);
+		}
+	)
+);
+// Set coming soon values
+$nfd_module_container->set(
+	'comingsoon',
+	array(
+		'option_name'         => 'mm_coming_soon',
```

Comment:
We should probably just have an upgrade routine to update this option.

Reply:
Now have an upgrade routine for this in place and migrating to use `nfd_coming_soon` in module and plugin.